www/public/css/application.css

```diff
@@ -1,238 +1,336 @@
-	margin-top: 0.5em;
+a.a--dead {
+	text-decoration: line-through;
 }
-div.fieldWithErrors {
-	border-left: 1em solid mistyrose;
-	margin-left: -1em;
+
+input.nav__input {
+	font-family: var(--font-sans-serif);
+	padding: 16px;
+	background-color: rgba(0, 0, 0, 0.05);
+	border-radius: 6px;
+	border: 2px solid transparent;
+	min-width: 256px;
+}
+@media (max-width: 1280px) {
+	form.nav__form,
+	input.nav__input {
+		display: block;
+		width: 100%;
+		min-width: 0px;
+	}
+}
+@media (max-width: 1280px) {
+	form.nav__form--header {
+		display: none;
+	}
 }
 
-#comments {
-	font-size: 90%;
-	margin-top: 1em;
+input.nav__input:focus {
+	outline: none;
+	border: 2px solid var(--red-primary);
+	background-color: #FFF;
 }
-div.comment span {
-	font-weight: bold;
+
+div.main__container {
+	border-radius: 6px;
 }
-div.comment pre {
-	font-family: monospace;
-	font-style: italic;
-	margin-right: 0.5em;
-	margin-left: 0.5em;
-	overflow: auto;
+
+footer.footer {
+	border-top: 1px solid var(--border);
+	padding-bottom: 64px;
+}
+
+hr.hr {
+	height: 1px;
+	border: none;
+	background-color: var(--border);
+	margin-top: var(--gap);
+	margin-bottom: var(--gap);
+	box-sizing: border-box;
+}
+
+h1.h1 {
+	font-size: 32px;
+	padding: var(--gap) var(--gap) 0 var(--gap);
+	font-weight: 300;
+	margin: 0;
+}
+@media (max-width: 960px) {
+	h1.h1 {
+		font-size: 24px;
+	}
+}
+
+h2.h2 {
+	font-size: 24px;
+	margin: var(--gap);
+	letter-spacing: -0.025em;
+	line-height: 1.35em;
+}
+@media (max-width: 960px) {
+	h2.h2 {
+		font-size: 20px;
+	}
+}
+
+h2.h2:first-child {
+	margin-top: var(--gap);
 }
 
-div.article {
-	margin-top: 0.5em;
+h2.h2:last-child {
+	margin-bottom: var(--gap);
 }
-div.article h3 {
-	font-weight: normal;
-	margin-bottom: 0;
+
+span.h2__span {
+	font-weight: 300;
+	letter-spacing: 0.025em;
 }
-div.article h3 b {
-	white-space: pre;
+
+h3.h3 {
+	font-size: 18px;
+	font-weight: 400;
+	margin: var(--gap) 0 0 0;
 }
 
-div.article div p {
-	margin: 0.5em;
+h3.h3:first-of-type {
+	margin-top: 0;
 }
-div.article div a {
-	font-style: italic;
+
+h4.h4 {
+	margin-top: var(--gap);
 }
 
-ul.plugins h3 {
-	font-weight: normal;
-	margin-bottom: 6px;
+p.p {
+	line-height: 1.35em;
 }
-ul.plugins p {
-	font-style: italic;
-	margin-top: 6px;
+
+ul.ul {
+	display: flex;
+	flex-direction: column;
+	row-gap: 12px;
+	padding: 0;
+	margin: 12px 0 0 0;
+	list-style: none;
+}
+
+ul.ul--grid {
+	padding-top: 16px;
+	border-top: 1px solid var(--border);
 }
 
-td {
-	font-size: 11px;
-	vertical-align: top;
+li.ul__li {
+	display: flex;
+	column-gap: 16px;
 }
-table thead tr th.cmAction, table tbody tr td.cmAction {
-	text-align: center;
-	width: 20px;
+@media (max-width: 960px) {
+	li.ul__li {
+		display: block;
+	}
 }
-table tbody tr td img { vertical-align: bottom; }
 
-table tbody tr.cmAction {
-	background-color: #FAFAFA;
-	border-top: 1px dotted brown;
-	border-bottom: 1px dotted brown;
-	color: black;
+li.ul__li--grid {
+	display: grid;
+	grid-template-columns: 1fr 3fr;
```

Comment:
There are longer plugin names that exceed the default width of the left-hand column:

<img width="892" alt="Screenshot 2022-04-17 at 10 16 29" src="https://user-images.githubusercontent.com/795488/163706464-a8f38e08-4c48-47fb-a21a-519451d70266.png">

Reply:
I ended up changing the column ratio from `1fr 3fr` to `1fr 2fr`, that gives it enough room for the current plugins name length (and a few more extra characters too).

If longer much names are expected in the future this component might need to be replaced as each row in the list it's own _little grid_ independent from the other ones.